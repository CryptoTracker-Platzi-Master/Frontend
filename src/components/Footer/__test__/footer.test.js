import React from "react";
import "@testing-library/jest-dom/extend-expect";
import { render } from "@testing-library/react";
import { Footer } from "../";

describe("[ component / Footer ]", () => {
  describe("when  `Footer` is mounted", () => {
    it("[`Footer`]", () => {
      const { asFragment } = render(<Footer />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
