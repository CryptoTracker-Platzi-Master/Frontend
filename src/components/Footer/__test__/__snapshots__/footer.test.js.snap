// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[ component / Footer ] when  \`Footer\` is mounted [\`Footer\`] 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <p
      class="footer--Copyright"
    >
      Copyright
    </p>
  </footer>
</DocumentFragment>
`;
